tests/fixtures.rs

```diff
@@ -51,3 +66,37 @@ pub fn include_with_context() {
 
     compare("include_with_context", &mut context);
 }
+
+#[test]
+pub fn graceful_fail_when_nonexistent_file() {
+    // TODO: Write this test
```

Comment:
I would name this test `error_on_nonexistent_file`, since I wouldn't say returning a failed `Result` is particularly graceful. It's debatable, I guess. Anyway, I think my name fits the behavior a bit better :)

This test shouldn't be hard to write, just pass a bogus path to `parse_file` and assert that the result is an error.